babel.config.js

```diff
@@ -4,7 +4,7 @@ const packageJSON = require(path.join(__dirname, 'package.json'))
 
 // RedwoodJS targets Node.js 12.x because this is the default version
 // for Netlify's functions.
-const TARGETS_NODE = '12.16'
+const TARGETS_NODE = '14.18.3'
```

Comment:
```suggestion
const TARGETS_NODE = '14.18'
```

Is it valid to omit the semver patch?

Reply:
We used to omit it so I imagine it is, but there should be an official answer somewhere; I'll try to find it.

Reply:
I couldn't find any docs but it looks like this is what babel does with the value: https://github.com/babel/babel/blob/e45d86c33380e2e7e98b5713443b8e20658494a9/packages/babel-helper-compilation-targets/src/utils.ts#L19-L37.

Basically, passing what we currently have '12.16' ends up being '12.16.0'. We should probably either pin the version or use legit semver, like '^14.18.3'.

Reply:
Well it looks like babel doesn't like `'^14.18.3'`: https://github.com/redwoodjs/redwood/runs/4932747479?check_suite_focus=true#step:9:21

Reply:
@thedavidprice whatever we pass goes through `semver.valid` which only seems to like something like `'14.18.3'`—something without any of the range specifiers. If we leave off the patch it's filled in for us with a `0`. 

I'd need to dig into babel more to understand the full implications, but one thing to keep in mind is that this isn't the the runtime version. It seems like that number is up to netlify, vercel, and to some degree, the user.